testing/tests/DevExpress.ui.widgets.editors/tagBox.tests.js

```diff
@@ -886,6 +886,52 @@ QUnit.module("the 'onValueChanged' option", moduleSetup, () => {
         assert.equal(valueChangeActionSpy.called, false, "onValueChanged was not fired");
     });
 
+    QUnit.test("onValueChanged should not be fired on first popup render (T838251)", function(assert) {
+        const data = [{
+            id: 1,
+            name: "First"
+        }, {
+            id: 2,
+            name: "Second"
+        }, {
+            id: 3,
+            name: "Third"
+        }];
+        const valueChangeActionSpy = sinon.spy();
+        const instance = $("#tagBox").dxTagBox({
+            dataSource: new DataSource({
+                store: new CustomStore({
+                    key: "id",
+                    load() {
+                        const res = $.Deferred();
+                        setTimeout(() => {
+                            res.resolve(data);
+                        }, TIME_TO_WAIT / 4);
+                        return res.promise();
+                    },
+                    byKey(key) {
+                        const res = $.Deferred();
+                        setTimeout(() => {
+                            res.resolve(key);
+                        }, TIME_TO_WAIT / 4);
+                        return res.promise();
+                    }
+                }),
+                paging: false
```

Comment:
```suggestion
                paginate: false
```
DataSource has no `paging` options. Did you mean [paginate](https://js.devexpress.com/Documentation/ApiReference/Data_Layer/DataSource/Configuration/#paginate)?

Reply:
I copied it from another example in this file